src/v2i-hub/PortDrayagePlugin/README.md

```diff
@@ -0,0 +1,73 @@
+## Introduction
+The Port Drayage Plugin in V2x-Hub facilitates infrastructure, vehicle and container handling equipment (CHE) communication for port drayage operations. The plugin provides and montior drayage actions for a freight truck inside and between a mock port and staging area. The list of actions includes ENTER_STAGING_AREA, PICKUP, EXIT_STAGING_AREA, ENTER_PORT, DROPOFF, PORT_CHECKPOINT, HOLDING_AREA, and EXIT_PORT.
+### Deployment Instructions
```

Comment:
No need to include V2X-Hub deployment instructions, just instructions specific to port drayage plugin